modules/05 ES6 and Typescript/Assignments/4/main.ts

```diff
@@ -0,0 +1,100 @@
+// exercise 23
+interface IStringUtills<T> {
+    getVowels(str: string): T;
+    removeChar(str: string, pos: T): string;
+    incrementLetters(str: string): string;
+}
+class StringUtils implements IStringUtills<number>{
+
+    public getVowels = (str: string): number => {
+        let count: number = 0;
+        let vowels: any = "aeiou";
```

Comment:
vowels type should be "string"

Reply:
I know. Just when i did a string type it somehow threw an error and asked to put vowels as any.